wiki/Tournaments/en.md

```diff
@@ -13,7 +13,7 @@ Official tournaments hosted by the Tournament Management.
 osu! World Cup is a country-based 4v4 Team VS tournament following double-elimination rule. It is an international event and is usually done yearly with the osu!communities around the world working together to make a tournament to create a competitive nature.
```

Comment:
`osu! World Cup` -> `The osu! World Cup`

`osu!communities` -> `osu! communities`